### PR TITLE
PHPDoc update

### PR DESCRIPTION
**Pull request by GawainLynch**

### CODE REVIEW

File: src/Storage/Repository.php [hunk 1]
```diff
@@ -6,6 +6,7 @@
 use Bolt\Events\StorageEvent;
 use Bolt\Events\StorageEvents;
 use Bolt\Storage\Entity\Builder;
+use Bolt\Storage\Entity\Entity;
```
Comment (bobdenotter) [inline src/Storage/Repository.php]: If it didn't break without the `use`, is it really needed, then?

Comment (GawainLynch) [inline src/Storage/Repository.php]: It gets the PHPDoc in-line so that when you're extending or typehinting, it doesn't show as an error in the IDE

Comment (bobdenotter) [inline src/Storage/Repository.php]: Ah, all good then!

**Comment by bobdenotter** [comment]
:+1:

**Pull request merged**